.docker/wp-config.php

```diff
@@ -0,0 +1,129 @@
+<?php
```

Comment:
`SHOULD` - The Meta Environment version might be good to look at to see if it has anything that's missing from here.

Reply:
Synced and added some missing configs in https://github.com/WordPress/wordcamp.org/pull/113/commits/1e3bc2e5e0ebcc4643db21c0d98a69b94a51b6c8